web/apps/labelstudio/src/components/Error/Error.scss

```diff
@@ -34,7 +34,9 @@
     margin: 15px 0;
   }
 
-  &__stracktrace {
+  &__stacktrace {
+    --font-family: monaco, monospace;
```

Comment:
This will affect the body definition based on this way of declaring, is that intended?

Reply:
nope not intended - i was intending that font to only apply to `.<block>__stacktrace` couldnt remember if updating the css var will have it apply to only this element or if it would bubble it up or it wouldnt work at all - was just trying to test this now actually.
will add `font-family: var(--font-family);`

Reply:
See my other suggestions, I figured this may have been the case and my suggestion is to bring to the top a variable for sans and mono separate, which then gets utilized across the entire codebase.